### ISSUE TITLE
Train Two Prediction Version

### ISSUE THREAD

**Issue by AnuragSahu** [AnuragSahu/3D-Point-Cloud-Object-Reconstruction-from-Monoclar-RGB-Image#3]
This Might take a while but it should be done within 5 Days, the mini tasks to be accomplished are 
1. Get the Training Code (https://github.com/AnuragSahu/CV_Project_2020_3D_Object_Reconstruction_from_a_Single_Image/issues/2)
2. Select the Classes (Approx 10) to train the data on
3. Get the Respective Dateset from the Shape-net
4. Train the Network and report the .log file
5. Also get the Training Loss Curve and Validation Loss Curve
